eng/config/PublishData.json

```diff
@@ -128,34 +128,24 @@
       "insertionTitlePrefix": "[d17.4]"
     },
     "release/dev17.5-vs-deps": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "vsBranch": "rel/d17.5",
-      "vsMajorVersion": 17,
-      "insertionCreateDraftPR": false,
-      "insertionTitlePrefix": "[d17.5p2]"
-    },
-    "main": {
       "nugetKind": [
         "Shipping",
         "NonShipping"
       ],
       "vsBranch": "main",
```

Comment:
Use main as the targeting branch for 17.5, since we are faster then the VS branch
Will change this back to 17.5 this weekend